RedacteurPortaal.Tests/Grains/ArchiveGrainTests.cs

```diff
@@ -41,10 +42,56 @@ public async Task CanAddArchiveCorrectly()
         Assert.Equal(toSaveArchive.MediaVideoItems, updatedArchive.MediaVideoItems);
         Assert.Equal(toSaveArchive.MediaPhotoItems, updatedArchive.MediaPhotoItems);
         Assert.Equal(toSaveArchive.Scripts, updatedArchive.Scripts);
+    }
 
+    [Fact]
+    public async Task CanGetASingularArchiveCorrectly()
+    {
+        var guid = Guid.NewGuid();
 
+        var toSaveArchive = new ArchiveModel(guid, "Title", "Label", new List<MediaPhotoItem> { }, new List<MediaVideoItem> { }, new List<MediaAudioItem> { }, new List<NewsItemModel> { }, DateTime.UtcNow, new List<string> { });
+        var archiveGrain = this._cluster.GrainFactory.GetGrain<IArchiveGrain>(guid);
+        await archiveGrain.CreateArchive(toSaveArchive);
 
+        var archiveInGrain = await this._cluster.GrainFactory.GetGrain<IArchiveGrain>(guid).Get();
 
+        Assert.NotNull(archiveInGrain);
```

Comment:
de return van Get() is nooit null, je moet de waardes van het object zelf controleren

Reply:
Object zelf wordt nu gecontroleerd.